### DISCUSSION TITLE
czml数据源怎么一键关闭卫星轨道路径

### DISCUSSION THREAD

**Discussion by j524275214** [zouyaoji/vue-cesium#213] [Q&A] [answered]
我想一键关闭图片中的卫星轨道路径
而不是viewer.dataSources._dataSources[0].entities.values[i].path.show = false
因为如果有成千上万颗卫星时 直接循环遍历会很消耗性能
有办法么 求教
![image](https://user-images.githubusercontent.com/40748734/173546324-63f8a268-ed80-4daa-b85f-c125c6fd7fce.png)

**Comment by zouyaoji** [comment, marked as answer]
vc-viewer上绑定shouldAnimate属性，并改为false。

或者直接用api
`viewer.clock.shouldAnimate = false`

**Reply by j524275214** [reply]
谢谢大佬 但是我需要的是隐藏path而非暂停

**Reply by zouyaoji** [reply]
数据源本身还有个show属性啊，用这个呗。

**Reply by j524275214** [reply]
还是这样吗viewer.dataSources._dataSources[0].entities.values[i].path.show = false

**Reply by zouyaoji** [reply]
用了组件的话 直接给组件绑定show来控制
https://zouyaoji.top/vue-cesium/#/zh-CN/component/datasources/vc-datasource-czml

**Reply by j524275214** [reply]
好的 我去研究一下 非常感谢

**Reply by HHANG52** [reply]
你好，还是这样单独的去循环去除轨迹显示吗？ 有别的方式吗？怎么解决的
